### ISSUE TITLE
Sort Institutes

### ISSUE THREAD

**Issue by afalenski** [SiLeBAT/mibi-portal-client#290]
As a MiBi-Portal user registering for an account I would like to select from an institute list sorted by alphabet to be able to find my institute faster.

Add placeholder text.

**Comment by afalenski** [comment]
Well sorted. Now, two entries which are obviously not institutes are shown when you click into the Institutes field:
"bunde, 12345 Berlin"
"bundes, 12345 Berlin"
New ticket will be created and should be done before the new release.
